### DISCUSSION TITLE
Cannot renew my iTwin

### DISCUSSION THREAD

**Discussion by ksurell** [iTwin/itwinjs-core#6077] [Q&A]
This might be a question for Bentley support but I'll give it a shot here as well.

I get mail saying that my iTwing are about to expire and I need to refresh them. When I try to that the Refresh option is grayed out.
Any ideas?

Best regards,
Krister Surell

![image](https://github.com/iTwin/itwinjs-core/assets/32495078/e3362c57-7359-47a4-8da7-96958a90eb9e)

**Comment by williamkbentley** [comment]
Hello - You need to have at least the Developer role assigned to you in order to do iTwin actions. An account administrator can assign the role

**Reply by ksurell** [reply]
I'm the administrator and I have refreshed them several times before.

**Reply by pratikshan85** [reply]
@ksurell Please go through the below document for you reference to add user as developer for iTwin Platform .
[User Roles.pdf](https://github.com/iTwin/itwinjs-core/files/12856199/User.Roles.pdf)

Also, follow the below link for Roles and permission related details.
https://developer.bentley.com/roles-and-permissions/

**Comment by ksurell** [comment]
Hi,

I do not have Account Settings in my profile.
[image: image.png]

This is what I see when I open Roles and Permissions.
[image: image.png]

I also have this info in my profile. I looks that it has expired but I in
my iTwin it says that I have 4 days left.
[image: image.png]

Den tis 10 okt. 2023 kl 12:54 skrev pratikshan85 ***@***.***>:

> @ksurell <https://github.com/ksurell> Please go through the below
> document for you reference to add user as developer for iTwin Platform .
> User Roles.pdf
> <https://github.com/iTwin/itwinjs-core/files/12856199/User.Roles.pdf>
>
> Also, follow the below link for Roles and permission related details.
> https://developer.bentley.com/roles-and-permissions/
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/iTwin/itwinjs-core/discussions/6077#discussioncomment-7239757>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHX5LZTCJFKRMOIEAB3YUBDX6USQBAVCNFSM6AAAAAA5WY7YG6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TEMZZG42TO>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

*Krister Surell*
Ringduvevägen 13
132 47 Saltsjö-Boo
Tfn: 070-315 10 75

**Comment by ksurell** [comment]
Hi,

Today my iTwing expired. Do you have an solution for this?

Best regards,
Krister Surell

Den tis 10 okt. 2023 kl 14:41 skrev Krister Surell ***@***.***
>:

> Hi,
>
> I do not have Account Settings in my profile.
> [image: image.png]
>
> This is what I see when I open Roles and Permissions.
> [image: image.png]
>
> I also have this info in my profile. I looks that it has expired but I in
> my iTwin it says that I have 4 days left.
> [image: image.png]
>
> Den tis 10 okt. 2023 kl 12:54 skrev pratikshan85 ***@***.***
> >:
>
>> @ksurell <https://github.com/ksurell> Please go through the below
>> document for you reference to add user as developer for iTwin Platform .
>> User Roles.pdf
>> <https://github.com/iTwin/itwinjs-core/files/12856199/User.Roles.pdf>
>>
>> Also, follow the below link for Roles and permission related details.
>> https://developer.bentley.com/roles-and-permissions/
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/iTwin/itwinjs-core/discussions/6077#discussioncomment-7239757>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AHX5LZTCJFKRMOIEAB3YUBDX6USQBAVCNFSM6AAAAAA5WY7YG6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TEMZZG42TO>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
>
> *Krister Surell*
> Ringduvevägen 13
> 132 47 Saltsjö-Boo
> Tfn: 070-315 10 75
>


-- 

*Krister Surell*
Ringduvevägen 13
132 47 Saltsjö-Boo
Tfn: 070-315 10 75
